### PR TITLE
Documentation for EnterpriseDB

### PR DESCRIPTION
**Pull request by rbellamy**
Home page, about, faq, navigation and an EnterpriseDB-specific page.

### CODE REVIEW

File: documentation/database/enterprisedb.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+layout: documentation
+menu: enterprisedb
+subtitle: EnterpriseDB
+---
+<div id="databaseEnterpriseDB">
+    <h1>EnterpriseDB</h1>
+    <h1><a name="EnterpriseDB"></a>Oracle<a href="#EnterpriseDB" class="section_anchor"></a></h1>
```
Comment (axelfontaine) [inline documentation/database/enterprisedb.html]: The Oracle reference still needs to be changed

File: documentation/database/enterprisedb.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+-- Placeholder
+INSERT INTO ${tableName} (name) VALUES (&#x27;Mr. T&#x27;);
+
+-- COPY ... FROM STDIN
+COPY copy_test (c1, c2, c3) FROM stdin;
+1	utf8: ümlaute: äüß	NaN
+2	\N	123
+3	text	123.234444444444449
+\.</pre>
```
Comment (axelfontaine) [inline documentation/database/enterprisedb.html]: Is this COPY syntax supported?

**Comment by axelfontaine** [comment]
Looks awesome. Please see my comments

**Comment by rbellamy** [comment]
Edits complete.

**Pull request merged**